app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+import shutil
+
+
+def copy_file(argument: str) -> tuple:
+    if len(argument.rsplit(" ")) == 3:
+        src_path = argument.rsplit(" ")[1]
+        dst_path = argument.rsplit(" ")[2]
```

Comment:
```suggestion
        cmd, source_path, destination_path = argument.split()
```
Preferred to use unzipping